File: SEMESTER 2/Java/Cycle 1/Fibonacci.java
import java.io.*;
class Fibonacci
{
	public static void main(String args[]) throws IOException
	{
		InputStreamReader ip = new InputStreamReader(System.in);
		BufferedReader br = new BufferedReader(ip);

		int i, n;
		int t1=0, t2=1;
		int nextTerm = t1 + t2;

		System.out.println("Enter number of terms: ");
		n = Integer.parseInt(br.readLine());

		System.out.print("Fibonacci series: " + t1 + " " + t2 + " ");

		for(i=3; i<=n; i++)
		{
			System.out.print(nextTerm + " ");
			t1 = t2;
			t2 = nextTerm;
			nextTerm = t1 + t2;
		}
	}
}

File: SEMESTER 3/PYTHON/fibonacci.py
# Set the number of terms in the Fibonacci series
n_terms = int(input("Enter the number of terms in the Fibonacci series: "))

# Initialize the first two terms of the series
first = 0
second = 1

# Print the first two terms
print(first)
print(second)

# Generate the remaining Fibonacci numbers
for i in range(2, n_terms):
    next_term = first + second
    print(next_term)
    first = second
    second = next_term
